Cargo.toml

```diff
@@ -15,4 +15,5 @@ serde = "1.0.152"
 thiserror = "1.0.38"
 
 [dev-dependencies]
+temp-env = "0.3"
```

Comment:
This avoids the need to unset environment variables as a few of the tests have done. As environment variables are per-process, there was also a risk that two tests could interfere with each other if they had similar `struct`s but different values.